app/admin/[organization]/page.tsx

```diff
@@ -14,26 +15,23 @@ export async function generateStaticParams() {
 }
 
 const EventPage = async ({ params }: { params: { organization: string } }) => {
-  const events: IEvent[] = await (
-    await fetch(`${apiUrl()}/organizations/${params.organization}/events`, {
-      cache: 'no-store',
-    })
-  ).json()
+  try {
+    const response = await axios.get(`${apiUrl()}/organizations/${params.organization}/events`)
```

Comment:
Adding Error handling is good, but it doesn't really solve the issue. 
We're calling an internal API here. What's causing the 500?